ignite/services/chain/init.go

```diff
@@ -126,7 +121,9 @@ func (c *Chain) InitAccounts(ctx context.Context, conf *chainconfig.Config) erro
 	c.ev.Send("🗂  Initialize accounts...")
 	c.ev.SendView(accounts)
 
-	_, err = c.IssueGentx(ctx, createValidatorFromConfig(conf))
+	if conf.Validators != nil {
+		_, err = c.IssueGentx(ctx, createValidatorFromConfig(conf))
+	}
```

Comment:
Maybe it would be good to have a comment describing the case where there are no validators because the default is to assume that a config is not valid if it doesn't have validators.